acceptance/framework/consul/cli_cluster.go

```diff
@@ -305,3 +303,26 @@ func (c *CLICluster) runCLI(args []string) ([]byte, error) {
 	cmd.Dir = config.CLIPath
 	return cmd.Output()
 }
+
+// mergeArgs merges the given args with the default args.
+func mergeArgs(defaults map[string]string, args ...string) []string {
+	contains := func(s []string, e string) bool {
+		for _, a := range s {
+			if a == e {
+				return true
+			}
+		}
+		return false
+	}
+
+	for k, v := range defaults {
+		if !contains(args, k) {
+			args = append(args, k)
+			if v != "" {
+				args = append(args, v)
+			}
+		}
+	}
+
+	return args
+}
```

Comment:
@jmurret what do you think about this?

Reply:
yea, i think that's about right.  Do you have an example of defaults?  I am just curious what these look like because you are appending the key and the value from default args....which if the k/v pair is something like enable/true, you will append "enable" and "true".  But if it is enable/"", then you only append "enable". Is that desired?

Reply:
👍